asf_welsh_energy_consultation/pipeline/process_data.py

```diff
@@ -17,26 +18,34 @@ def get_enhanced_mcs():
     Returns:
         pd.DataFrame: Dataset as described above.
     """
-    mcs = get_mcs_domestic()
-    og = get_offgas()
-    countries = get_countries()
-    rural = get_rurality()
+    mcs = get_data.get_mcs_domestic()
+    og = get_data.get_offgas()
+    countries = get_data.get_countries()
+    rural = get_data.get_rurality()
 
     # join with off-gas data
     mcs = mcs.merge(og, on="postcode", how="left")
     mcs["off_gas"] = mcs["off_gas"].fillna("On gas").replace({True: "Off gas"})
 
     # join with regions in order to filter to Wales
     mcs = mcs.merge(countries, on="postcode", how="left")
+    if mcs.country.isna().sum() > 0:
+        logger.warning(
+            f"{mcs.country.isna().sum()} MCS installation records have no country match."
+            f"Potential loss of data when filtering for Wales."
+        )
```

Comment:
I love these warnings! Very insightful to have them. Would just add either a space of a \n line break - the "match.Potential" on the console freaks me out because it looks like code 😂😂😂
```suggestion
        logger.warning(
            f"{mcs.country.isna().sum()} MCS installation records have no country match. "
            f"Potential loss of data when filtering for Wales."
        )
```